File: NotificationSystem/NotificationSystem.Web/Controls_Search/ctrSearch_Admin.ascx.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.UI;
using System.Web.UI.WebControls;


	public partial class ctrSearch_Admin : System.Web.UI.UserControl
	{
        private string m_UserName = string.Empty;

        public string UserName
        {
            get
            {
                return m_UserName;
            }
            set
            {
                m_UserName = value;
            }
        }
        protected void Page_Load(object sender, EventArgs e)
		{

		}
        public void PopulateSearchControl()
        {
            m_UserName = txtUserName.Text;
        }

    }

File: NotificationSystem/NotificationSystem.Web/Controls_Search/ctrSearch_Admin.ascx.vb
Imports System
Imports System.Web.UI

Namespace NotificationSystem.NotificationSystem.Web.Controls_Search
    Public Partial Class ctrSearch_Admin
        Inherits UserControl
        Private m_UserName As String = String.Empty

        Public Property UserName As String
            Get
                Return m_UserName
            End Get
            Set(value As String)
                m_UserName = value
            End Set
        End Property
        Protected Sub Page_Load(ByVal sender As Object, ByVal e As System.EventArgs) Handles Me.Load
            If Not IsPostBack Then
                txtUserName.Text = m_UserName
            End If

        End Sub
        Public Sub PopulateSearchControl()
            m_UserName = txtUserName.Text
        End Sub

    End Class
End Namespace
